### PR TITLE
Improve example for spread

### PR DESCRIPTION
**Pull request by justsml**
`foo` examples leave a lot to imagination.

But `fetch` might need to link to promises section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,12 +184,11 @@ getFinalPrice(500); // 850
...
 
 ```javascript
-function foo(x, y, z) {
-    console.log(x, y, z);
-}
-
-let arr = [1, 2, 3];
-foo(...arr); // 1 2 3
+let args = ['https://github.com/metagrover/ES6-for-humans/', {method: 'GET'}];
+fetch(...args)
```
Comment (metagrover) [inline README.md]: While I agree that the foo examples leave a lot to imagination, this example with fetch can get a bit tricky for the first timers to grasp.

It'd be best if we can use a collection, ex:

```js
const ingredients = ['bread', 'butter', 'jam'];
```

and use it in a function (say, `makeBreakfast`) - using the spread operator.

**Review by metagrover — CHANGES_REQUESTED** [review]
Hi @justsml, Overall example is better than the existing one, but needs to be simplified for the ease of readers.

File: README.md (hunk 1 above)
Comment (justsml) [reply README.md]: great idea Deepak! 

Does something like this work?

```js
const makeToast = (breadType, topping1, topping2) => {
  return `I had ${breadType} toast with ${topping1} and ${topping2}`;
};

const ingredients = ['sourdough', 'avocado', 'kale'];
makeToast(...ingredients); 
// "I had sourdough toast with avocado and kale"
```

**Comment by justsml** [comment]
Hey @metagrover - thanks for your feedback! 

How do my changes look?

**Review by metagrover — APPROVED** [review]

**Comment by metagrover** [comment]
Thanks for contributing @justsml :100:

**Pull request merged**